data/yaml/name_variants.yaml

```diff
@@ -361,6 +361,11 @@
 - canonical: {first: Noushin Rezapour, last: Asheghi}
   variants:
   - {first: Noushin, last: Rezapour Asheghi}
+- canonical: {first: Iñaki, last: Lacunza}
+  orcid: 0009-0005-3305-9070
+  institution: Computer Vision Center
```

Comment:
This isn't really an institution, but a department at an institution. Please find the name of the institution of highest degree.

Reply:
ping @Azax4